### PR TITLE
Update teamcollision.txt

### PR DESCRIPTION
**Pull request by cravenge**
Correct Windows offset for CGameMovement::PlayerSolidMask

### CODE REVIEW

File: extra/teamcollision.txt [hunk 1]
```diff
@@ -13,7 +13,7 @@
 			"CGameMovement::PlayerSolidMask"
 			{
 				"linux"		"16"
-				"windows"	"17"
+				"windows"	"15" // 17 points to CGameMovement::CalcRoll
```
Comment (shqke) [inline extra/teamcollision.txt:16]: Must leave out a misleading comment. Comment on issue usually goes into pull request.

**Review by shqke — CHANGES_REQUESTED** [review]

File: extra/teamcollision.txt (hunk 1 above)
Comment (cravenge) [reply extra/teamcollision.txt]: Ah sorry, I am still trying to get used to GitHub. I did a quick vtable scanning and found out that the current offset for that function was directing to another function.

File: extra/teamcollision.txt [hunk 2]
```diff
@@ -13,7 +13,7 @@
 			"CGameMovement::PlayerSolidMask"
 			{
 				"linux"		"16"
-				"windows"	"17"
+				"windows"	"15" // was 17 (pointing to CGameMovement::CalcRoll)
```
Comment (shqke) [inline extra/teamcollision.txt:16]: I apologize, you've misunderstood my meaning, a gamedata file of project's source code is not a place for comments on changes, all of comments go into "conversation" tab on a github website. 😀

Basically, you need to remove your comment from the file completely, leaving changed offset value within double quotes as the only edit.

**Review by shqke — CHANGES_REQUESTED** [review]

File: extra/teamcollision.txt (hunk 1 above)
Comment (shqke) [reply extra/teamcollision.txt]: Yes, usually windows offsets's value go by -1 from linux, but I've missed this one out when I was correcting others @ 3eb88ece06eb2cf237b967a85a4eff97c0102ef1.

File: extra/teamcollision.txt (hunk 2 above)
Comment (cravenge) [reply extra/teamcollision.txt]: Ah okay, I got it.

**Review by shqke — APPROVED** [review]

**Comment by shqke** [comment]
Thanks for PR!

**Pull request merged**